### DISCUSSION TITLE
Issue on sorting by price on configurable products

### DISCUSSION THREAD

**Discussion by merdiano** [bagisto/bagisto#6326] [General]
# Bug Report

Sorting by price is not working properly on configurable products with different variant prices.

## Bagisto Version

- [x ] master branch
- [ x] 1.4.2
- [ ] 1.3.2
- [ ] 1.3.1
- [ ] 1.3.0

## Steps To Reproduce

Add configurable products with different variant prices
sort by price
## Actual Result

products are not sorted in ascending or descending order

**Comment by devansh-webkul** [comment]
Hi @merdiano,

Have you reproduced this in the master version?

**Comment by merdiano** [comment]
Yes @devansh-webkul i have reproduced in the master also.

**Comment by Sanjaybhattwebkul** [comment]
@merdiano 
I am not getting any issues with sorting products by price. Please find the screenshot [Here](https://prnt.sc/iMQNo5lENPDJ).

**Comment by Sanjaybhattwebkul** [comment]
@merdiano 
Is it possible to share a screen recording? for the same.

**Comment by merdiano** [comment]
It is on storefront
https://sarga.com.tm/kadn/giyim-kadn/ayal-egin-esik-sweatshirt?mode=list&order=desc&sort=price&sort=price&order=asc

**Comment by Sanjaybhattwebkul** [comment]
@merdiano 
Is the **APP_URL**  is set in **.env** file ?

**Comment by merdiano** [comment]
shure it is set to domain name

**Comment by Sanjaybhattwebkul** [comment]
Hi @merdiano 
Have you done any customization?

**Comment by merdiano** [comment]
Yes. I have some customizations within my own package. But not in webkul package. Did not customize velocity theme and Product repository. Today i found that sorting by price in descending order works fine but ascendin order is not working when there is simple products and configruable products in same list.

**Comment by merdiano** [comment]
Filter by price on configurable products also has issues when the lowest min price is 0
?price=0,50 not working
?price=1,50 working

**Comment by Amitk-webkul** [comment]
Hi @merdiano, I believe you've found your solution. If you have any more questions in the future, don't hesitate to reach out. Just go ahead and close this discussion for now.
